test/test-mda-models/aissemble-test-data-delivery-pyspark-model-basic/pyproject.toml

```diff
@@ -33,7 +33,7 @@ urllib3 = "^1.26.18"
 black = ">=22.1.0"
 behave = ">=1.2.6"
 nose = ">=1.3.7"
-testcontainers = ">=3.7.1"
+testcontainers = ">=4.5.1"
```

Comment:
(This is not related to this PR/Issue) but it would be nice if we could inject this version as a variable across the test module, since I feel like it's easy to miss updating our `test` module